### ISSUE TITLE
AV8B Harrier Edit function

### ISSUE THREAD

**Issue by MadogPL** [Santi871/DCSWaypointEditor#63]
Could there be an option to not ADD new waypoints in Harrier ( now it goes: add waypoint->77->that creates realistically next waypoint) but EDIT/OVERRIDE them in order?
It make a lot of confusion in combined flights when first waypoint of harrier flight is eight in a row :) (for example)

I'd gladly contribute for this feature to be available.

**Comment by DarKFeneR996** [comment]
Harrier now has a bug: before u had to push Odu POS button to go on Lat or Lon coords. Now once u push enter, it automatically enters between Lat and Long. So before it put E or W or N or S, u have to push again Odu POS Button

I'm trying to recompile it to fix these problems and set that script as u asked but I'm having warning :(

**Comment by MadogPL** [comment]
Hallo! Indeed, after ENT it goes automatically for long. Isn't that "just" one less button hit to be scripted as it assumes it switched into E-W?

**Comment by DarKFeneR996** [comment]
Hello, I reported this bug to 51st-Vfw FORK. It has no problems in that fork. Yep, that was just 1 less button to push

**Comment by MadogPL** [comment]
So, it it a bug, or a feature? Because right now WE does not work for Harrier (it types gibberish). Not sure how it suppose to be.

**Comment by DarKFeneR996** [comment]
This version is bugged cuz it enters, automatically changes coords type (from Lon to Lat) when pressing enter, then the program repress to change coords type, so it bug the input.
So, before update (maybe 2 months ago)
Input 8 (N) or 2 (S), 00'00'00, ENTER
Pess POS from ODU buttons to switch into Lon
4 (W) or 6 (E), 000'00'00, ENTER
Pess POS from ODU buttons to switch into Lat

Actually:
Input 8 (N) or 2 (S), 00'00'00, ENTER
*automatically switchea into Lon*
4 (W) or 6 (E), 000'00'00, ENTER
*automatically switches into Lat*

NOTE: For Lon coordinate, you need 7 numbers instead of 6 as Lat

NOTE: If you Press N or S while in Lon input, it stills blank, until u press the right one (W or E)

**Comment by DarKFeneR996** [comment]
So, to answer you: this version of the application has the bug; the game has been updated, not bugged

**Comment by MadogPL** [comment]
Hallo! Indeed, after ENT it goes automatically for long. Isn't that "just" one less button hit to be scripted as it assumes it switched into E-W?

**Comment by MadogPL** [comment]
Soooo... will there be an update of WE to new updated game?

And would it be possible to also edit, like suggested in my original topic, to edit the waypoint, not add them?

**Comment by DarKFeneR996** [comment]
To edit instead of create new, is already possible, in game manually and with the program. But this one is stuck without any update

**Comment by MadogPL** [comment]
may I ask how with Waypoint Editor we can edit existing waypoints? It has right now command "77->ENT".

I'm very sorry, but I don't understand your answer. For me right now:
Because of the automatic lon/lat change of a game, Waypoint Editor does one extra unnecessary step, ending with wrong waypoints added.

What is "stuck without any update"? Let's get this into points, because I have a feeling that we are running circles.

1. Issue of not working anymore, because of automatic lon/lat change.
2. Function to edit exiting waypoints, instead of adding them.

**Comment by DarKFeneR996** [comment]
This is working.
https://github.com/51st-Vfw/DCSWaypointEditor
You can ask there to improve the waypoint selection instead of 77
